sdk/search/tests.yml

```diff
@@ -11,6 +11,7 @@ extends:
       Public:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
         ServiceConnection: azure-sdk-tests
+        Location: eastus
```

Comment:
Any reason for specifying the location rather than letting the deployment script determine it?

Reply:
@alzimmermsft This "can" be removed. The reason it is there is because if Live Tests are creating too many resources the quota limit can be reached for a particular region and the region will need to be moved. I specified eastus due to it having no other deployments. (per our conversation with with Ben)